src/org/jitsi/impl/neomedia/stats/MediaStreamStats2Impl.java

```diff
@@ -206,13 +217,24 @@ else if (direction == StreamDirection.UPLOAD)
      */
     public void updateRtt(long ssrc, long rtt)
     {
-        // TODO(boris) Currently we only maintain an RTT value for the entire
-        // MediaStream, and not for any of the individual SSRCs. At the time of
-        // this writing, it is unclear to me whether it should be kept this way,
-        // or whether keeping a per-SSRC value can be useful for something. So
-        // I am doing the easier thing.
+        // RTT value for the entire MediaStream
         receiveStats.setRtt(rtt);
         sendStats.setRtt(rtt);
+
+        // RTT value for individual SSRCs
+        // skip invalid ssrc
+        if (ssrc < 0)
+            return;
+
+        // directly get the receive/send stats to avoid creating unnecessary
+        // stats
+        ReceiveTrackStatsImpl receiveStat = receiveSsrcStats.get(ssrc);
+        if (receiveStat != null)
+            receiveStat.setRtt(rtt);
+
+        SendTrackStatsImpl sendStats = sendSsrcStats.get(ssrc);
+        if (sendStats != null)
+            sendStats.setRtt(rtt);
```

Comment:
Can we use consistent naming (e.g. receiveSsrcStat, sendSsrcStat) and add braces?